providers/apache/beam/pyproject.toml

```diff
@@ -57,20 +57,21 @@ requires-python = ">=3.10"
 # After you modify the dependencies, and rebuild your Breeze CI image with ``breeze ci-image build``
 dependencies = [
     "apache-airflow>=2.10.0",
-    'apache-beam>=2.60.0',
-    "pyarrow>=16.1.0",
-    "numpy>=1.26.0",
-
+    'apache-beam>=2.60.0; python_version < "3.13"',
+    "pyarrow>=16.1.0; python_version < '3.13'",
+    "numpy>=1.22.4; python_version<'3.11'",
+    "numpy>=1.23.2; python_version<'3.12' and python_version>='3.11'",
+    "numpy>=1.26.0; python_version>='3.12' and python_version < '3.13'",
```

Comment:
Should we be adding for 3.13 too?

Reply:
Nope. Beam is excluded from 3.13 for now. We should likely change some deps here when work on it's compatibility.